sass/shoppable_gallery.scss

```diff
@@ -420,13 +416,13 @@
       .yotpo-shoppable-tagged-products-title {
         font-family: Arial;
         color: #888888;
-        font-size: 18px;
+        font-size: 16px;
         font-weight: 400;
         letter-spacing: 0.13px;
         line-height: 44.619px;
         text-align: left;
         transform: scaleX(1.0083);
-        margin-left: 20px;
+        padding-left: 2.5%;
```

Comment:
@livnekes up above there is .yotpo-shoppable-stats which has padding of 2.5%, and this element should be aligned to it.